src/main/java/org/osiam/storage/query/OsiamAntlrErrorListener.java

```diff
@@ -42,55 +42,52 @@
 
     /**
      * {@inheritDoc}
-     * <p/>
+     * <p>
      * The method is overridden to throw an exception if the parsed input has an invalid syntax.
      */
     @Override
     public void syntaxError(Recognizer<?, ?> recognizer, @Nullable Object offendingSymbol, int line,
-            int charPositionInLine, String msg, @Nullable RecognitionException e) {
+                            int charPositionInLine, String msg, @Nullable RecognitionException e) {
 
-        String errorMessage = msg;
-        if (e instanceof InputMismatchException && msg.endsWith(" expecting VALUE")) {
```

Comment:
Since I'm registering the `OsiamErrorListener` for both the parser and the lexer the error is no longer triggered by the `InputMissmatchException`.